compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/scenario/Conversions.scala

```diff
@@ -24,7 +24,21 @@ final class Conversions(
   private val packageIdSelf: PackageIdentifier =
     PackageIdentifier.newBuilder.setSelf(empty).build
 
-  private val coidToNodeId = ledger.ledgerData.coidToNodeId
+  // The ledger data will not contain information from the partial transaction at this point.
+  // We need the mapping for converting error message so we manually add it here.
+  private val ptxCoidToNodeId = machine.ptx.nodes.toList
+    .collect({
+      case (nodeId, node: N.NodeCreate.WithTxValue[V.ContractId]) =>
+        node.coid match {
+          case acoid: V.AbsoluteContractId =>
+            (node.coid.asInstanceOf[V.AbsoluteContractId], ledger.ptxNodeId(nodeId))
```

Comment:
```suggestion
            acoid -> ledger.ptxNodeId(nodeId)
```

Reply:
:facepalm:, forgot to change this once I switched to a case statement. Thanks!